bundles/ui/org.eclipse.smarthome.ui/src/main/java/org/eclipse/smarthome/ui/internal/chart/ChartServlet.java

```diff
@@ -135,6 +132,16 @@ protected void deactivate() {
         httpService.unregister(SERVLET_NAME);
     }
 
+    protected void modified(Map<String, Object> config) {
```

Comment:
Is modified always called and you do not need to to process the config in activate() at all? I didn't know this, that makes the implementation indeed easier :-)